examples/non_fittable_mixin.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+
+import numpy as np
+
+from neuraxle.base import BaseStep, NonTransformableMixin, NonFittableMixin
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.misc import BaseCallbackStep, TapeCallbackFunction
+
+
+def main():
+    # Fit method is automatically implemented as changing nothing.
```

Comment:
There could be more comments in the test to explain the why and what and how things are happening

Reply:
(could do this later).